src/armTemplates/resources/appServicePlan.ts

```diff
@@ -0,0 +1,63 @@
+import { ArmResourceTemplateGenerator } from "../../models/armTemplates";
+import { ServerlessAzureConfig, ResourceConfig } from "../../models/serverless";
+
+export const AppServicePlanResource: ArmResourceTemplateGenerator = {
+  getTemplate: () => {
+    return {
+      "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
```

Comment:
how did you get this starter template?  we should make sure to document where this come front, how to get updated one etc.

Reply:
In azure portal - when you are about to deploy a new resource there is an option in the UI to view the template.  I grabbed from there and modified.